From hjk@linutronix.de  Wed Apr 15 16:18:40 2009
From: "Hans J. Koch" <hjk@linutronix.de>
Date: Sat, 11 Apr 2009 04:18:25 +0200
Subject: UIO: fix specific device driver missing statement for depmod
To: Andrew Morton <akpm@linux-foundation.org>
Cc: hjk@linutronix.de, Greg KH <greg@kroah.com>
Message-ID: <20090411021813.GA3651@local>
Content-Disposition: inline


On Fri, Apr 10, 2009 at 01:50:50PM -0700, Andrew Morton wrote:
> On Fri, 10 Apr 2009 13:32:01 GMT
> bugzilla-daemon@bugzilla.kernel.org wrote:
> 
> > http://bugzilla.kernel.org/show_bug.cgi?id=13059

drivers/uio/uio_cif.c misses a MODULE_DEVICE_TABLE, this fixes it.

Signed-off-by: Hans J. Koch <hjk@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/uio/uio_cif.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/uio/uio_cif.c
+++ b/drivers/uio/uio_cif.c
@@ -147,5 +147,6 @@ static void __exit hilscher_exit_module(
 module_init(hilscher_init_module);
 module_exit(hilscher_exit_module);
 
+MODULE_DEVICE_TABLE(pci, hilscher_pci_ids);
 MODULE_LICENSE("GPL v2");
 MODULE_AUTHOR("Hans J. Koch, Benedikt Spranger");
